Tools/Scripts/webkitdirs.pm

```diff
@@ -2844,6 +2844,16 @@ sub generateBuildSystemFromCMakeProject
         $ENV{"CXXFLAGS"} = "-m32" . ($ENV{"CXXFLAGS"} || "");
         $ENV{"LDFLAGS"} = "-m32" . ($ENV{"LDFLAGS"} || "");
     }
+    if (architecture() eq "arm64" && shouldBuild32Bit()) {
+        # CMAKE_LIBRARY_ARCHITECTURE is needed to get the right .pc
+        # files in Debian-based systems, for the others
+        # CMAKE_PREFIX_PATH will get us /usr/lib, which should be the
+        # right path for 32bit. See FindPkgConfig.cmake.
+        push @cmakeArgs, '-DFORCE_32BIT=ON -DCMAKE_PREFIX_PATH="/usr" -DCMAKE_LIBRARY_ARCHITECTURE=armv7-a+fp';
```

Comment:
I was suspicious of the `-DCMAKE_PREFIX_PATH=/usr" ([documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html)) but I guess this is OK. If it works, it's probably fine.

I wonder if this would accidentally override jhbuild if using build-webkit in jhbuild mode, which would be worth testing. But I also think that mode should go away....

Reply:
Yeah, I just copied what was already there :ship:, but it seems to work